domain/model/response_api.go

```diff
@@ -0,0 +1,15 @@
+package model
+
+// Structure to parse the character data
```

Comment:
Please consider giving more details as this could be a response for any object unless it would be for general purpose.